fuzz/src/onion_message.rs

```diff
@@ -165,37 +193,41 @@ mod tests {
 		let logger = TrackingLogger { lines: Mutex::new(HashMap::new()) };
 		super::do_test(&::hex::decode(one_hop_om).unwrap(), &logger);
 		{
 			let log_entries = logger.lines.lock().unwrap();
 			assert_eq!(log_entries.get(&("lightning::onion_message::messenger".to_string(),
-						"Received an onion message with path_id None and no reply_path".to_string())), Some(&1));
+						"Received an onion message with path_id None and a reply_path".to_string())), Some(&1));
+			assert_eq!(log_entries.get(&("lightning::onion_message::messenger".to_string(),
+						"Responding to onion message with path_id None".to_string())), Some(&1));
+			assert_eq!(log_entries.get(&("lightning::onion_message::messenger".to_string(),
+						"Failed responding to onion message with path_id None: TooFewBlindedHops".to_string())), Some(&1));
```

Comment:
We have an assumption because of the secret key in `do_test` that the handling node is the second node (i.e., node[1]), so this is the only realistic reply path. In all the other test cases, the handling node isn't the recipient node, so it wouldn't trigger the responding code path.